WordPressKitTests/WordPressOrgXMLRPCApiTests.swift

```diff
@@ -200,8 +200,16 @@ class WordPressOrgXMLRPCApiTests: XCTestCase {
                 XCTAssertEqual(error.domain, WPXMLRPCFaultErrorDomain)
                 // 403 is the 'faultCode' in the HTTP response xml.
                 XCTAssertEqual(error.code, 403)
-                XCTAssertNil(error.userInfo[WordPressOrgXMLRPCApi.WordPressOrgXMLRPCApiErrorKeyData as String])
                 XCTAssertNil(error.userInfo[WordPressOrgXMLRPCApi.WordPressOrgXMLRPCApiErrorKeyStatusCode as String])
+
+                // The change highlights one difference between the existing Alamofire-backed API and the new
+                // URLSession-backed API: the error returned by the new one may has HTTP response body which may not
+                // be the case exist in the old API. I think this is an acceptable change.
```

Comment:
```suggestion
                // URLSession-backed API: the error returned by the new one has an HTTP response body which is not
                // the case in the old API. I think this is an acceptable change.
```

I think it's an acceptable change, too. In particular because the new way has more information than the old. If it was the other way around, then one would have to consider if the loss of information is dangerous for the client. But luckily that's not our case 😄